File: 1328. Break a Palindrome/Solution.java
class Solution {
    public String breakPalindrome(String palindrome) {
        if (palindrome.length() < 2) {
            return "";
        }
        char[] s = palindrome.toCharArray();
        for (int i = 0; i < s.length / 2; i++) {
            if (s[i] != 'a') {
                s[i] = 'a';
                return new String(s);
            }
        }
        s[s.length - 1] = 'b';
        return new String(s);
    }
}

File: 1328. Break a Palindrome/Solution.py
class Solution:
    def breakPalindrome(self, palindrome: str) -> str:
        if len(palindrome) < 2:
            return ""
        for i, c in enumerate(palindrome[: len(palindrome) // 2]):
            if c != "a":
                return palindrome[:i] + "a" + palindrome[i + 1 :]
        return palindrome[:-1] + "b"
